sagemaker-triton/ensemble/tf-dali-ensemble-cv.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
<p>How are these magic numbers (crop, mean, std) determined? How could a customer know how to change them if they're bringing a different dataset or specifications?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
added comments in the pipelines definition